.github/workflows/ci.yaml

```diff
@@ -24,13 +24,11 @@ jobs:
       - name: Run linters
         run: poetry run poe lint
 
-  test:
+  test-build-only:
```

Comment:
the only tests that run are API retry tests. This essentially runs zero tests on MacOS and Windows, which is kinda sketchy, but at least it verifies building the package...